package.json

```diff
@@ -48,7 +48,7 @@
     "react-dom": "^15.6.1",
     "react-flexbox-grid": "^1.1.3",
     "react-redux": "^5.0.4",
-    "react-router-redux": "^4.0.8",
+    "react-router-redux": "^5.0.0-alpha.6",
```

Comment:
Risky. I like it!

Reply:
We're at the cutting edge in metrics (at least until Monday)